File: se226Lab1/time_logic.py

x=input("Enter a large integer:\n")
hour= int(x)//3600
minutes= (int(x)-hour*3600)//60
seconds= (int(x)-hour*3600-minutes*60)
print(x+"Seconds is "+ str(hour)+" hours, "+str(minutes)+" minutes, "+str(seconds)+" seconds")

File: se226Lab1/time_logic.cpp
#include <iostream>
using namespace std;

int main(){
int x;
cout << "Enter a large int: ";
   cin >> x;

   int h = x/3600;
   int m = (x-h*3600)/60;
   int s = (x-h*3600-m*60);

   cout << x<< " second is "<< h<<" hour, "<<m<<" minute, "<<s<<" second";
   return 0;
   
}